### PR TITLE
Fix for #21909: Error message 'Flashlight listener registration failed  for listener class'  is found in server.log when EJB monitoring is HIGH

### PR DESCRIPTION
**Pull request by srinivaskrishnan**
Fix for #21909: Error message 'Flashlight listener registration failed  for listener class'  is found in server.log when EJB monitoring is HIGH

### CODE REVIEW

**Comment by srinivaskrishnan** [comment]
@glassfishrobot Run CI tests please

**Comment by srinivaskrishnan** [comment]
@glassfishrobot Run CI tests please

File: appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/BaseContainer.java [hunk 1]
```diff
@@ -4801,6 +4801,7 @@ protected void createMonitoringRegistry() {
 	String appName = null;
 	String modName = null;
 	String ejbName = null;
+    boolean isMonitorRegistryMediatorCreated = false;
```
Comment (yaminikb) [inline appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/BaseContainer.java]: Fix format

File: appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/StatefulSessionContainer.java [hunk 2]
```diff
@@ -513,34 +513,53 @@ protected void loadCheckpointInfo() {
...
                 containerInfo.ejbName);
         cacheProbeListener.register();
 
+        if (isHAEnabled) {
+            sfsbStoreMonitor = new HAStatefulSessionStoreMonitor();
+        } else {
+            sfsbStoreMonitor = new StatefulSessionStoreMonitor();
+        }
+        sessionBeanCache.setStatefulSessionStoreMonitor(sfsbStoreMonitor);
+        _logger.log(Level.FINE, "[SFSBContainer] registered monitorable");
```
Comment (yaminikb) [inline appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/StatefulSessionContainer.java]: Add isLoggable check

**Review by yaminikb — CHANGES_REQUESTED** [review]

**Comment by srinivaskrishnan** [comment]
@glassfishrobot Run CI tests please

**Review by yaminikb — APPROVED** [review]

**Pull request merged**